apitools/base/py/base_api.py

```diff
@@ -398,6 +399,14 @@ def max_retry_wait(self, value):
                 'max_retry_wait must be a postiive integer')
         self.__max_retry_wait = value
 
+    @property
```

Comment:
If there's no validation or anything here, you can just make this a public property.